<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>todoApp</class>
 <widget class="QMainWindow" name="todoApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>todoApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="todayLabel">
            <property name="text">
             <string>Today Tasks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="todayTask">
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="pendingLabel">
            <property name="text">
             <string>Pending Tasks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="pendingTask">
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="finishedLabel">
          <property name="text">
           <string>Finished Tasks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="finishedTask">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semibold</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoScroll">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Task"/>
    <addaction name="actionDelete_Tasks"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionToday_Task"/>
    <addaction name="actionPending_Task"/>
    <addaction name="actionFinished_Tasks"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_todoApp"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Task"/>
   <addaction name="actionToday_Task"/>
   <addaction name="actionPending_Task"/>
   <addaction name="actionFinished_Tasks"/>
   <addaction name="actionDelete_Tasks"/>
  </widget>
  <action name="actionNew_Task">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new-icon.png</normaloff>:/new-icon.png</iconset>
   </property>
   <property name="text">
    <string>New &amp;Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/quit-icon.png</normaloff>:/quit-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionToday_Task">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/today-task.png</normaloff>:/today-task.png</iconset>
   </property>
   <property name="text">
    <string>Today &amp;Tasks</string>
   </property>
  </action>
  <action name="actionPending_Task">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Pending-icon.png</normaloff>:/Pending-icon.png</iconset>
   </property>
   <property name="text">
    <string>Pending &amp;Tasks</string>
   </property>
  </action>
  <action name="actionDelete_Tasks">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/delete-icon.png</normaloff>:/delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>Delete Tasks</string>
   </property>
  </action>
  <action name="actionFinished_Tasks">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/finished-task.png</normaloff>:/finished-task.png</iconset>
   </property>
   <property name="text">
    <string>Finished &amp;Tasks</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAbout_todoApp">
   <property name="text">
    <string>About &amp;TodoApp</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
